src/im/keyboard/compose.cpp

```diff
@@ -170,11 +170,14 @@ bool ComposeState::typeImpl(KeySym sym, std::string &result) {
 
     reset();
     // Compose success or key should be ignored.
-    if (composeResult->empty()) {
```

Comment:
shouldn't require anymore. The all branch in the previous if check the null-ness.